README.md

```diff
@@ -2,6 +2,8 @@
 
 This model is used to predict the risk of West Nile Virus being present two consecutive weeks in a row. 
 
+Most of the work in this project is data wrangling to get the data into a format where each row represents the results for a week / location collection, and the interval between the dates is regular (i.e. a week apart). Once this is accomplished, a model is fit that is very similar to a generalized linear model to predict the likelihood that WNV will be present if it was present last week. This allows our teams to go out and spray for mosquitos a week earlier than they would have otherwise.
```

Comment:
I think this should be more direct and focus on the outcome of the project. Something like:

> This project estimates the probability of a subsequent mosquito testing positive for West Nile Virus (WNV) when a mosquito has already been found to have WNV at a particular trap. The model allows the Chicago Department of Public Health to proactively react to the spread of WNV. A multilevel (hierarchical) regression to predict the likelihood that WNV will be present if it was present last week. A cross-validation study has shown that this model correctly predicts consecutive results for WNV virus 80 percent of the time.
>
> This repository provides the source code used to generate the predictions. However, it can be entirely executed outside of the City of Chicago computer network. Though, it does allow the statistical methodology to be reviewed, evaluated, and replicated elsewhere.